application/app/Rules/PictureValidation.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class PictureValidation implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $size_mb = $value->getSize() / 1024 / 1024;
```

Comment:
sizeだけだとなんの？となるので名前工夫したほうがベターです
（クラス名がPictureなので、おそらく画像のことだろうなーとは予測できますが、あくまで予想だけなので）

```suggestion
        $size_mb = $value->getSize() / 1024 / 1024;
```
```suggestion
        $file_size_mb = ...;
        $picture_size_mb = ...;
```